dymaptic.Pro.ZoomToCoordinates/Views/ShowAbout.cs

```diff
@@ -0,0 +1,26 @@
+﻿using ArcGIS.Desktop.Framework;
+using ArcGIS.Desktop.Framework.Contracts;
+
+namespace dymaptic.Pro.ZoomToCoordinates.Views
+{
+    internal class ShowAbout : Button
+    {
+
+        private About? _about = null;
+
+        protected override void OnClick()
+        {
+            //already open?
+            if (_about != null)
+                return;
+            _about = new About
+            {
+                Owner = FrameworkApplication.Current.MainWindow
+            };
+            _about.Closed += (o, e) => { _about = null; };
```

Comment:
Hmm, this _could_ cause a memory leak, we are registering a handler on each click, but never actually unregistering it. Setting the `_about` to null might be killing the handler, but it might not. I would 
1. replace the lambda with a named private method in this class, so adding the handler is `_about.Closed += MyHandlerMethod;`, and
2. in the new method, call `_about.Closed -= MyHandlerMethod;` before setting to `null`.

Reply:
Created private OnAboutClosed event handler and explicitly unsubscribe to before setting to null.